### DISCUSSION TITLE
is it posable to invert pressure sensitivity?

### DISCUSSION THREAD

**Discussion by nagaisa** [xournalpp/xournalpp#4215] [General]
The stylus i use seams to decrease pressure the harder i press. Would it be portable to invert it? or add a way to invert it? (either system or just the app)

**Comment by hyperupcall** [comment]
It sounds like a system solution will help solve your problem, rather than something application-specific, which will cause troubles once you switch between apps (and complicate this app too much). It would definitely be possible to invert it, but the difficulty would depend on the device.

Here is a [thread](https://github.com/linuxwacom/input-wacom/issues/157) on linuxwacom - it specifies lateral sensitivity, but maybe you can accomplish something similar for pressure sensitivity. You would definitely have to do more specific research
